libmachine/provision/suse.go

```diff
@@ -205,3 +182,34 @@ func (provisioner *SUSEProvisioner) Provision(swarmOptions swarm.Options, authOp
 	err = provisioner.Service("docker", serviceaction.Enable)
 	return err
 }
+
+func (provisioner *SUSEProvisioner) configureFirewall() error {
+	tcpPorts := "22 80 443 2376 2379 2380 6443 9099 9796 10250 10254 30000:32767"
+	udpPorts := "8472 30000:32767"
+	cmds := make([]string, 0)
+	if _, installed := provisioner.SSHCommand("rpm -q firewalld"); installed == nil {
+		tcpPorts := strings.ReplaceAll(tcpPorts, ":", "-")
+		udpPorts := strings.ReplaceAll(udpPorts, ":", "-")
+		for _, port := range strings.Split(tcpPorts, " ") {
+			cmds = append(cmds, fmt.Sprintf("sudo firewall-cmd --permanent --add-port=%s/tcp", port))
+		}
+		for _, port := range strings.Split(udpPorts, " ") {
+			cmds = append(cmds, fmt.Sprintf("sudo firewall-cmd --permanent --add-port=%s/udp", port))
+		}
+		cmds = append(cmds, "sudo firewall-cmd --reload")
```

Comment:
These requirements are mirrored in https://rancher.com/docs/rancher/v2.x/en/installation/options/firewall/